stations.csv

```diff
@@ -28703,7 +28703,7 @@ id;name;slug;uic;uic8_sncf;latitude;longitude;parent_station_id;country;time_zon
 30855;Rioseras;rioseras;;;42.4537965;-3.6398029;;ES;Europe/Madrid;t;f;f;t;t;f;;;f;;f;;f;ezt3t0;t;;f;;f;;f;;f;;f;;f;;f;;;;f;;f;;f;;f;;f;;f;f;;;;;;;;;;;;;;;;;;;;urn:trainline:public:nloc:csv30855;
 30856;Riovena;riovena;;;43.2472396;-6.5832063;;ES;Europe/Madrid;t;f;f;t;t;f;;;f;;f;;f;ezeq40;t;ESOYB;t;;f;;f;;f;;f;;f;;f;;;;f;;f;;f;;f;;f;;f;f;;;;;;;;;;;;;;;;;;;;urn:trainline:public:nloc:csv30856;
 30857;Rípodas;ripodas;;;42.6864006;-1.3060756;;ES;Europe/Madrid;t;f;f;t;t;f;;;f;;f;;f;ezx4f6;t;;f;;f;;f;;f;;f;;f;;f;;;;f;;f;;f;;f;;f;;f;f;;;;;;;;;;;;;;;;;;;;urn:trainline:public:nloc:csv30857;
-30858;Ripoll;ripoll;;;42.20064;2.19033;;ES;Europe/Madrid;t;f;f;t;t;f;;;f;;f;;f;sp981f;t;ESOEL;t;;f;;f;;f;;f;;f;;f;;;;f;;f;;f;;f;;f;;f;f;;;;;;;;;;;;;;;;;;;;urn:trainline:public:nloc:csv30858;
+30858;Ripoll;ripoll;7177200;;42.195536;2.195977;;ES;Europe/Madrid;f;f;f;f;t;f;;;f;;f;;f;sp981f;t;ESOEL;t;;f;;f;;f;;f;;f;;f;;;;f;;f;;f;;f;;f;;f;f;;;;;;;;;;;;;;;;;;;;urn:trainline:public:nloc:csv30858;
```

Comment:
I think that this is the city station. I'm not sure that we necessarily want to convert it to the Cercanias station?

Reply:
I think the same comment applies to `Montgat`.

Reply:
This should be fine, these locations currently don't do anything so it seems that we may as well convert them into something useful

Reply:
I think that we are returning some journeys when searching the city station so we might want to consider that.

Reply:
This should be fixed now

Reply:
I think that `is_suggestible` is false for this station - do we want to stop suggesting it?